wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt

```diff
@@ -56,15 +90,31 @@ fun getIcon(entity: Entity<Map<String, Any>>?, domain: String, context: Context)
                 CommunityMaterial.Icon2.cmd_light_switch
             }
             "input_button" -> CommunityMaterial.Icon2.cmd_gesture_tap_button
+            "input_datetime" -> CommunityMaterial.Icon.cmd_calendar_clock

```

Comment:
This one changes [depending on state](https://github.com/home-assistant/frontend/blob/ffc4ca5b568ad56754967462e9dca01bae2bd70d/src/common/entity/domain_icon.ts#L105-L111) in the frontend